base/server/src/main/java/com/netscape/cms/realm/PKIPostgreSQLRealm.java

```diff
@@ -121,42 +121,6 @@ public void initInternal () throws LifecycleException {
         }
     }
 
-    /**
-     * This method will create the tables if they do not exist.
-     */
-    public void setup() throws Exception {
-
-        logger.info("Setting up PostgreSQL realm");
-
-        String filename = "/usr/share/pki/acme/realm/postgresql/create.sql";
-        String content = new String(Files.readAllBytes(Paths.get(filename)));
-
-        String[] stats = content.split(";");
-        for (String sql : stats) {
-            sql = sql.trim();
-            if (StringUtils.isEmpty(sql)) continue;
-            logger.info("SQL: " + sql);
-
-            try (PreparedStatement ps = connection.prepareStatement(sql)) {
-                ps.executeUpdate();
-
-            } catch (SQLException e) {
-
-                // https://www.postgresql.org/docs/current/errcodes-appendix.html
-                String sqlState = e.getSQLState();
-
-                // If table already exists, ignore
-                if (!"42P07".equals(sqlState)) {
-
-                    logger.error("Unable to set up PostgreSQL realm: " + e.getMessage());
-                    logger.error("SQL state: " + sqlState);
-
-                    throw e;
-                }
-            }
-        }
-    }
```

Comment:
Is it possible to keep this code? I think it was meant to make ACME easier to use, i.e. just run PostgreSQL and ACME, and it's ready to use (ACME doesn't require initial users). If necessary (e.g. for EST) the tables can be created first and this code will not alter the tables. We probably can add a new param in `realm.conf` to specify the path, for example:
```
create=/usr/share/pki/acme/realm/postgresql/create.sql
```

Reply:
The problem of this code is that it check for the table and if not present then it will create the tables. If the DB is provided by the user and the table names do not match the new table should not be created. We can make this optional.